test/unit/formidable.test.js

```diff
@@ -159,6 +160,92 @@ function makeHeader(filename) {
     form.emit('end');
   });
 
+  describe(`${name}#_onPart`, () => {
+    describe('when not allow empty files', () => {
+      describe('when file is empty', () => {
+        test('emits error when part is received', (done) => {
+          const form = getForm(name, {
+            multiples: true,
+            allowEmptyFiles: false,
+          });
+
+          const part = new Stream();
+          part.mime = 'text/plain';
+          // eslint-disable-next-line max-nested-callbacks
+          form.on('error', (error) => {
+            expect(error.message).toBe(
+              'options.allowEmptyFiles is false, file size should be greather than 0',
+            );
+            done();
+          });
+          form.onPart(part);
+          part.emit('end');
+          expect.assertions(1);
```

Comment:
I think this assertion can't be reached because of the `done()` that ends the test and proceeds. Not very sure.

I still didn't look at the error logs though. I guess errors are not related to this PR anyway because dependency update PRs are also breaking without any changes to the source code in the last few months.

Reply:
Yeah, in this scenario using the `done` callback was not necessary. Removed on [d3f8b31](https://github.com/node-formidable/formidable/pull/662/commits/d3f8b3174888a6734188ebf829184a468ca93f40).